website/config.yaml

```diff
@@ -66,13 +66,13 @@ params:
         url: 'https://slack.k8s.io/'
         icon: fab fa-slack
         desc: 'Chat with us on Slack in the #aws-provider channel'
-  latest_release_version: v0.13.2
+  latest_release_version: v0.14.0-rc.0
   versions:
+    - "v0.14.0-rc.0"
```

Comment:
We could potentially leave `latest_release_version` as `v0.13.2` and keep all redirects to that version too and only add `v0.14.0-rc.0` to the dropdown.

Reply:
Agreed! Perhaps not even cut new docs and just let users rely on preview?

Reply:
Yeah I can definitely go either way on this. Let me change.

Reply:
Alright so if you were to manually go to `karpenter.sh/v0.14.0-rc.0/` the docs will show, but we won't redirect you there, and it's not in our dropdowns. You also get the warning that it's not the latest stable release which works out

Reply:
Beautiful